src/Grphp/Client/Interceptors/Base.php

```diff
@@ -89,6 +89,24 @@ public function getFullyQualifiedMethodName(): string
         return $stub->getServiceName() . '/' . ucfirst($methodName);
     }
 
+    /**
+     * Gets the fully qualified service name, e.g. grphp.test.Things
+     *
+     * @return string
+     * @throws StubNotFoundException
+     */
+    public function getServiceName(): string
+    {
+        $method = $this->getMethod();
```

Comment:
You don't need to call this outside the `if (empty($stub))` block

Reply:
updated